collection-scripts/gather_crs

```diff
@@ -10,9 +10,11 @@ fi
 # Resource list
 crs=()
 
-# explictly adding rabbit in the grep because it's not part of the openstack
-# umbrella and we rely on the upstream rabbitmq-cluster-operator
-for i in $(/usr/bin/oc get crd | grep -E "(openstack|rabbitmq)\.(org|com)" | awk '{print $1}')
+# Explictly adding rabbit in the grep because it's not part of the openstack
```

Comment:
the rabbitmq operator is actually part of the umbrella in the sense that we: install it, and create the rabbit cluster as part of OpenStackControlplane. I guess you just mean the CRD is named with a different domain?

Reply:
I actually didn't write the sentence about rabbitmq, I just made the "E" a capital letter and added a "." to make it into a sentence. I'm not sure what exactly the original author meant. The change, which comes with this PR is the second sentence about monitoring.rhobs.

Reply:
I changed the message as requested